### ISSUE TITLE
[Table]: Interactive contents of the table are not announced by screen reader

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#9984]
### Describe the bug

Interactive contents of the table are not announced by screen reader. When user navigates from one row to another, only header text is announced by not the interactive content

### Isolated Example

https://stackblitz.com/edit/github-xfs7ff?file=src%2FToDos.module.css,src%2Fmain.tsx,src%2FToDos.tsx

### Reproduction steps

1. Navigate to https://stackblitz.com/edit/github-xfs7ff?file=src%2FToDos.module.css,src%2Fmain.tsx,src%2FToDos.tsx
2. Start Voiceover on Mac or Jaws on windows
3. Navigate through the table and check the announcement

### Expected Behaviour

Row content should be announced with the column headers

### UI5 Web Components for React Version

1.29.8

### UI5 Web Components Version

1.24.4

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-core,
could you take a look at this issue? I am able to reproduce the problem in  https://stackblitz.com/edit/github-7aqd1q?file=main.js,index.html
Regards,
Todor

**Comment by ilhan007** [comment]
Hi @lateefsofi.

the Table v2 in the main package was developed exactly to overcome many a11y issues of the legacy v1 Table from the compat package. We have it for a long time, likely you migrated already. If not - this is recommended to do.
